recipes/imagl/all/conanfile.py

```diff
@@ -1,7 +1,12 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Version
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, patch, copy
+from conan.errors import ConanInvalidConfiguration
+import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.59"
 
 
 class ImaglConan(ConanFile):
```

Comment:
Right!!! What a mistake!

Reply:
Sorry, I messed up there by writing the `lgpl` part of the license in lowercase. It should probably be uppercase, i.e. `LGPL-3.0-only`.

Reply:
Thanks, I've just made the change. I hope that this is what blocks CI build with 'Invalid JSON String"...
By the way, I've changed source urls to github ones since my gitlab instance is temporarily down 😢